alibi_detect/saving/schemas.py

```diff
@@ -905,6 +905,7 @@ class LearnedKernelDriftConfig(DriftDetectorConfig):
     batch_size_predict: int = 1000000
     preprocess_batch_fn: Optional[str] = None
     epochs: int = 3
+    num_workers: int = 4
```

Comment:
Same comment here and a bit further on `num_workers = 4`.

Reply:
Updated default to `num_workers=0` in ffe1b78